src/components/LoanCards/LoanCardImage.vue

```diff
@@ -63,13 +63,40 @@ export default {
 		openInNewTab: {
 			type: Boolean,
 			default: false
+		},
+		imageEnhancementExperimentVersion: {
+			type: String,
+			default: ''
+		},
+		loanImageHash: {
+			type: String,
+			default: ''
 		}
 	},
+
 	computed: {
```

Comment:
Computed properties shouldn't be named 'get'; you can just call these imageRetinaUrl and imageStandardUrl

Reply:
Or maybe something like formattedRetinaImageUrl since retinaImageUrl is already a property.